pr_hero.md

```diff
@@ -1,2 +1,3 @@
 # Pull Request Heros
-Jeff Hollister
\ No newline at end of file
+Jeff Hollister
+Rhody R. Stats
```

Comment:
Looks great.  But be even better if we have the list of names as an unorder list.  Can you take care of that?